python/pyspark/ml/tests/test_feature.py

```diff
@@ -1269,12 +1270,25 @@ def test_rformula_string_indexer_order_type(self):
         )
         rf = RFormula(formula="y ~ x + s", stringIndexerOrderType="alphabetDesc")
         self.assertEqual(rf.getStringIndexerOrderType(), "alphabetDesc")
-        transformedDF = rf.fit(df).transform(df)
+        model = rf.fit(df)
+        transformedDF = model.transform(df)
```

Comment:
how about checking if rf.uid and model.uid are equal?

Reply:
SG